scripts/s3-logrotate/s3addblogfilerollover.sh

```diff
@@ -57,7 +57,7 @@ echo
 echo "Rotating addb files in each $s3_daemon_working_dir<s3server-instance> directory"
 if [[ -n "$s3_daemon_working_dir" && -d "$s3_daemon_working_dir" ]]
 then
-   # get s3server instance directories from /var/motr
+   # get s3server instance directories from /var/log/seagate/motr
```

Comment:
we can use $s3_daemon_working_dir instead of /var/log/.... in the comment. If any change in the path later on, will save us not to update this comment.